### ISSUE TITLE
What does your development environment look like?

### ISSUE THREAD

**Issue by sindresorhus** [wesbos/ama#1]
Machine, OS, editor, terminal, themes, fonts, etc.

**Comment by wesbos** [comment]
Macbook pro 15" retina

Sublime Text is my editor of choice - I've done two stints with both Vim and Atom. I learned a ton from Vim and I'll talk about Atom in another post.  I've got a pretty popular theme + color scheme for sublime text called [Cobalt2](https://github.com/wesbos/cobalt2) which pairs nicely with the Inconsolata font. Although the comments on [this post](http://wesbos.com/programming-fonts/) are making me want to switch.

For my terminal I run iTerm2.  I've got a [Cobalt 2](https://github.com/wesbos/cobalt2-iterm) theme and prompt that I use as well. I'm a huge fan of ZSH + Z to keep up command line productivity. I did a little video series on it → <http://commandlinepoweruser.com/>

Other than that, it depends on the type of project I'm running. I do a lot of Node so I like the MongoHub desktop client.  For LAMP stuff I use MAMP + sequel pro although I'm looking at moving to vagrant as its getting a little long in the tooth. 

Other than that I use Text Expander and Cloudapp  an insane amount - very helpful for writing.
